MSMetaEnhancer/libs/Annotator.py

```diff
@@ -49,6 +49,8 @@ async def annotate(self, spectra, jobs, repeat=False):
                         warning.add_info(exc)
                     except ServiceNotAvailable:
                         warning.add_warning(ServiceNotAvailable(f'Service {job.converter} not available.'))
+                    except Exception as e:
```

Comment:
I think it would be good to catch more information such as which spectrum object was used and which conversion - or is this captured by the logger itself intrinsically?

Also it would be good to record a specific keyword to the logger so that the log can be searched if errors of this type occurred.

Reply:
It is definitely assigned to the particular spectrum intrinsically, I will make sure the conversion and a keyword are there.

Reply:
Covered in [247036d](https://github.com/RECETOX/MSMetaEnhancer/pull/91/commits/247036d5f4bf025fa8fccf8cb01be33b8e566152). Errors in log are now annotated by particular `Exception` (e.g. `InvalidAttributeFormat`, `TargetAttributeNotRetrieved`). That means if the error is a general one (not recognised), the key word you are looking for is just `Exception`.